cmake/CommonStdlibBuiltins.cmake

```diff
@@ -125,6 +125,8 @@ function (generate_stdlib_or_target_builtins func ispc_name CPP_LIST BC_LIST)
             # Not all targets have 64bit
             disp_target_stdlib(${func} ${ispc_name} neon-i32x4 64 ${os} ${CPP_LIST} ${BC_LIST})
             disp_target_stdlib(${func} ${ispc_name} neon-i32x8 64 ${os} ${CPP_LIST} ${BC_LIST})
+            disp_target_stdlib(${func} ${ispc_name} neon-i8x16 64 ${os} ${CPP_LIST} ${BC_LIST})
+            disp_target_stdlib(${func} ${ispc_name} neon-i16x8 64 ${os} ${CPP_LIST} ${BC_LIST})
```

Comment:
It looks like now we can move them to the loop above like this:
```cmake
disp_target_stdlib(${func} ${ispc_name} ${target} 64 ${os} ${CPP_LIST} ${BC_LIST})
```

Reply:
Fixed